ksql-execution/src/main/java/io/confluent/ksql/execution/windows/KsqlWindowExpression.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2018 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.execution.windows;
+
+import com.google.errorprone.annotations.Immutable;
+import io.confluent.ksql.parser.Node;
+import io.confluent.ksql.parser.NodeLocation;
+import io.confluent.ksql.serde.WindowInfo;
+import java.util.Optional;
+
+@Immutable
+public abstract class KsqlWindowExpression extends Node {
```

Comment:
Unfortunately this is showing up as a new class, but it's just `KsqlWindowExpression` from ksql-parser moved to `ksql-execution` and with the interface that builds the aggregated `KTable` removed.